semantic-conventions/syntax.md

```diff
@@ -178,15 +178,12 @@ The following is only valid if `type` is `event`:
 
 The following is only valid if `type` is `metric`:
 
-- `metrics`, an optional list of metrics that belong to the semantic convention.
-  Each individual metric has the following semantics: 
-  - `id`, the ID of the metric. The fully qualified name of the metric includes its parent 
-    semantic convention prefix like so: `{parent.prefix}.{metric.id}`. 
-  - `brief`, a brief description of the metric.
-  - `instrument`, the [instrument type]( https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-types) 
-  that *should* be used to record the metric.
-  - `units`, the units in which the metric is measured, which should adhere to 
-     [UCUM](https://ucum.org/ucum.html). 
+  - `metric_name`, the metric name as described by the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#timeseries-model). 
+  - `instrument`, the [instrument type]( https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument) 
+  that *should* be used to record the metric. These include `counter`, `gauge`, `updowncounter`, and `histogram`. Whether the instrument 
+  is synchronous or asynchronous is an implementation detail and should not be specified. 
```

Comment:
I feel the link before (https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-types) was great in clarifying why we only list the sync instrument types and now we are missing this. 

What about we write something like this instead? (to replace the text starting at `These include `counter`...`)

```
Note that the semantic conventions must be written using the synchronous instrument types
 (`counter`, `gauge`, `updowncounter` and `histogram`). For more details:
 [Metrics semantic conventions - Instrument types](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-types)
```

Or

```
Note that the semantic conventions must be written using the synchronous instrument types
 (`counter`, `gauge`, `updowncounter` and `histogram`). Whether implementations of these
conventions choose the synchronous or asynchronous instrument types 
is considered an implementation detail detail and should not be specified as part of these conventions.
```

Reply:
Yeah good point, I agree. I like the first example you gave since it links back to the source of truth. 

I've gone ahead and used that, with a slight modification/clarification: "Note that the semantic conventions must be written using **the names of** the synchronous instrument types".

I wanted to clarify that we are using the _names_ of the instrument types, it doesn't mean the implementation has to use the synchronous type. I think that makes it slightly clearly. Besides that, everything is the same as your example! Thanks for that 😄